ttnn/cpp/ttnn/operations/moreh/moreh_dot_op_backward/moreh_dot_backward.hpp

```diff
@@ -14,9 +14,9 @@ struct MorehDotBackward {
         std::optional<const Tensor> other_grad,
         const std::optional<MemoryConfig> &mem_config);
 };
-}
+}  // namespace ttnn::operations::moreh::moreh_dot_backward
 
 namespace ttnn {
-constexpr auto moreh_dot_backward =
-    ttnn::register_operation<"ttnn::moreh_dot_backward", ttnn::operations::moreh::moreh_dot_backward::MorehDotBackward>();
+constexpr auto moreh_dot_backward = ttnn::
+    register_operation<"ttnn::moreh_dot_backward", ttnn::operations::moreh::moreh_dot_backward::MorehDotBackward>();
```

Comment:
```suggestion
    register_operation_with_auto_launch_op<"ttnn::moreh_dot_backward", ttnn::operations::moreh::moreh_dot_backward::MorehDotBackward>();
```

Reply:
This change is made here https://github.com/tenstorrent/tt-metal/pull/12993
This PR is only for revising moreh_dot

Reply:
Got it. Please handle this in a separate PR.